docs/remote/troubleshooting.md

```diff
@@ -1068,6 +1072,26 @@ Extensions that access locally attached devices will be unable to connect to the
 
 ## Questions and feedback
 
+### Reporting quality issues
+
+If you run into an issue with one of the remote development extensions, it's important to collect the correct logs so that we'll be able to help [diagnose your issue](https://aka.ms/vscode-remote/issues/new).
+
+We recommend turning on trace level telemetry to get the most detailed logs: `F1` -> `Developer: Set Log Level...` -> `Trace`. From here, the type of logs depends on the context of your issue.
```

Comment:
Each of the extensions comes with a log level setting (the core log level is not shared).

Reply:
I see - so does setting trace log level not matter, since the log level is taken care of once the user selects their specific output (i.e. "Remote-Containers: Show Log" will display the same info regardless of whether a user selected trace)?

Or, does setting trace level still matter, but users need to select this while connected to their remote (i.e. I need to be connected to a dev container, then set trace level, then can Remote-Containers: Show Log)?

Reply:
The first - extensions don't know about vscode's log level.

Reply:
Thanks for the explanation. I've removed the section about trace level telemetry since it doesn't seem helpful or relevant for reporting most issues here.